DigitalOffice/Services/UserService/RestRequests/EditUserRequest.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace LT.DigitalOffice.UserService.RestRequests
+{
+    public class EditUserRequest
```

Comment:
If a method for editing a user is needed for an ordinary user, then it is strange to change the mail and password here, because we have planned separate methods for this. Who is this endpoint for?

Reply:
Not only for ordinary user but for admin also.

Reply:
I think you didn't understand my previous message. I wanted to ask, is this endpoint needed for a regular user? But apparently this method is more likely for a conditional administrator.

Reply:
This method is not suitable for an ordinary user, the password should not be changed at the click of a finger

Reply:
(maybe there is a solution here) [link](https://docs.fluentvalidation.net/en/latest/custom-validators.html)

Reply:
I also have to check if a user which calls this method has Right #1 _or isAdmin_. Sooo it's for user and admin both. We just can hide "changing" password or mail for user there but unhide it for an admin.